cobalt/BUILD.gn

```diff
@@ -13,6 +13,53 @@
 # limitations under the License.
 
 group("gn_all") {
+  testonly = true
+
   # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  deps = [
+    ":cobalt",
+    "//starboard($starboard_toolchain)",
+  ]
+}
+
+if (is_android) {
+  import("//build/config/android/config.gni")
+}
+
+if (is_android) {
+  group("cobalt") {
+    testonly = true
+    deps = [ "//content/shell/android:content_shell_apk" ]
+  }
+} else {
+  executable("cobalt") {
+    testonly = true
+
+    sources = [
+      "cobalt.cc",
+      "cobalt_content_browser_client.cc",
```

Comment:
Can we enable these cobalt client implementations for Android too? I think Android can share these?

Reply:
Sure, that would make a lot of sense to do, but this PR doesn't include an Android target.